src/backend/src/appjl.jl

```diff
@@ -150,7 +167,15 @@ end
     return quantile(flatten(vcat(diffs...)), quantiles)
 end
 
-@memoize function var_stats_1d(domain,variable,quantiles=(0,1))
+"""
+    var_stats_1d(domain, variable, quantiles)
+
+Collect all values for `variable` from all scenarios in `domain` into a vector
+and return the quantiles of that vector.
+
+This is a bit expensive, so it is memoized.
+"""
```

Comment:
```julia
[I] ~/p/gcvt (cleanup|✚1) $ make back
cd src/backend && julia --project=. src/appjl.jl
ERROR: LoadError: cannot document the following expression:

#= /home/olie/projects/gcvt/src/backend/src/appjl.jl:178 =# @memoize function var_stats_1d(domai
n, variable, quantiles)
        #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:178 =#
        #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:180 =#
        vars = []
        #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:181 =#
        if domain == "od_matrices"
            #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:182 =#
            vars = [get(v, variable, []) for (k, v) = mats]
        elseif #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:183 =# domain == "links"
            #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:184 =#
            vars = [get(v, Symbol(variable), []) for (k, v) = links]
        end
        #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:186 =#
        (vcat(vars...) |> flatten) |> (x->begin
                    #= /home/olie/projects/gcvt/src/backend/src/appjl.jl:186 =#
                    quantile(x, quantiles)
                end)
    end

'@memoize' not documentable. See 'Base.@__doc__' docs for details.
```

I think there must be a bug in Memoize.jl but I can't reproduce it in a file on its own.

```
"""
    x(a)

This is a docstring
"""
@memoize function x(a)
    return rand(a)
end

x(1)
```

in this file does throw a similar error.